cache-filter/src/filter/http.rs

```diff
@@ -161,8 +161,47 @@ impl Context for CacheFilter {
         match self.get_http_call_response_body(0, body_size) {
             Some(bytes) => {
                 match Authorization::from_str(std::str::from_utf8(&bytes).unwrap()) {
-                    Ok(_response) => {
-                        // Handle cache hit here
+                    Ok(Authorization::Ok(response)) => {
```

Comment:
Just a heads up: this will change in the 0.8 release of the library, although it will likely have minimal impact.